src/elements/datepicker/datepicker/datepicker.ts

```diff
@@ -236,13 +279,19 @@ class SbbDatepickerElement<T = Date> extends LitElement {
   }
 
   private _attachInput(): void {
-    const input = findInput(this, this.input);
+    const input = findInput(this, this.input, 'input,sbb-date-input');
     if (this.inputElement === input) {
       return;
     } else if (this.inputElement) {
       this._datePickerController?.abort();
       this._inputObserver?.disconnect();
     }
+    const isNativeInput = !isDateInput(input);
+    if (isNativeInput && import.meta.env.DEV) {
+      console.warn(
```

Comment:
not sure whether we should wait with a warning? what if there is a blocker coming up for the new logic (doesn't hope so ;). but then they would have to change back.
the warning has the character of it should be fixed soonish for me.

Reply:
Wait for what? Either we commit or we don't. Or what is your consideration?